models/articles.models.js

```diff
@@ -0,0 +1,14 @@
+const db = require("../db/connection");
+
+function selectArticleById(article_id) {
+    return db.query(`SELECT * FROM articles WHERE article_id = $1`, [article_id])
+    .then(({rows}) => {
+      if (rows.length === 0) {
+        return Promise.reject({ status: 404, message: "article does not exist" });
+      } else {
+        return rows[0];
+      }
+    });
+  }
+
```

Comment:
Good logic here to deal with injection risk, handle non-existent articles and remove the article from the array